net-misc/gallery-dl/Manifest

```diff
@@ -0,0 +1 @@
+DIST gallery-dl-1.15.0.tar.gz 323180 BLAKE2B 2b0bac0e0a3a3c1c3cab1860f1f584fd09e5f6163a69a94419f2924bd62e1fb9464f8139f6cdce4a7020accaaa44cf1e60c03dab2db7d9568d35693a8f935c4e SHA512 3d15ebeaa088130efb4c33ad2bc2d7ed2af78eafc68ef00b451d65db7f6b589cf10e248c309677dbbfe7202885d602afdbcab330dae0ebce190bd1f5dce25834
```

Comment:
Can you update this to latest version? Got my eyes on it now so should be a fast-merge if you do.

Reply:
will do